assets/script.js

```diff
@@ -33,7 +33,11 @@ generateJokeEl.addEventListener('click', function(){
 
 const drinkButtonVariable = document.getElementById('drinkButton');
 function generateRecipe() {
-    let randomUrl = `https://www.thecocktaildb.com/api/json/v1/1/random.php`;
+    let select = document.getElementById('drink');
+    var liquor = select.options[select.selectedIndex].value;
+    console.log(liquor);
+    let randomUrl = `www.thecocktaildb.com/api/json/v1/1/filter.php?i=` + liquor;
```

Comment:
https:// - before the url and should work

Reply:
done